src/test/java/gov/usgs/wma/waterdata/springinit/BaseIT.java

```diff
@@ -49,4 +64,51 @@ public void assertJsonEquals(String expected, String actual) {
 			fail("Unexpected JSONException during test", e);
 		}
 	}
+
+	/**
+	 * A yyyy-MM-dd formated date, offset by the spec'ed number of days.
+	 *
+	 * @param offset Positive for future days, negative for past days, zero for today.
+	 * @return A yyyy-MM-dd formated date.
+	 */
+	public static String getDateStr(int offset) {
+		LocalDate now = LocalDate.now();
+		return now.plusDays(offset).format(DateTimeFormatter.ofPattern("yyyy-MM-dd"));
+	}
+
```

Comment:
These new static routines are general in nature.  Seems more appropriate for them to be in a util class or 
such and not in BaseIT

Reply:
It seems like BaseIt is a good common class used for testing.  It could be moved into another class just to contain these, but it wouldn't gain us much in our testing structure.  I'm willing to move these out if you feel strongly, but IMHO, it doesn't seem like much would be gained.

Reply:
I'm going to mark this as resolved.  Speak up if you really want this changed.

Reply:
good point, I am fine leaving them in for now.

Reply:
I would argue they might not be appropriate to be static and definitely should not bleed outside the test packages at this time.